client/flutter/lib/pages/home_page.dart

```diff
@@ -282,7 +233,11 @@ class _MenuGrid extends StatelessWidget {
                       _VerticalSpacer(),
                       IntrinsicHeight(
                         child: _MenuButton(
-                          onTap: () => _openQuestionsAndAnswers(context),
+                          onTap: () => logAnalyticsEvent('QuestionsAnswered'),
+                          page: QuestionIndexPage(
+                            dataSource: QuestionData.yourQuestionsAnswered,
+                            title: S.of(context).homePagePageButtonQuestions,
+                          ),
                           color: Color(0xff3DA7D4),
```

Comment:
Same comment as colour above

Reply:
Since these lines were not modified by this PR, they should not be refactored in this PR.